site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -248,6 +248,7 @@ The following tables lists the configurable parameters of the Agones chart and t
 
 | Parameter                                           | Description                                                                                     | Default                |
 | --------------------------------------------------- | ----------------------------------------------------------------------------------------------- | ---------------------- |
+| `agones.allocator.resources`                        | Allocator pods resource requests/limit                                                          | `{}`                   |
```

Comment:
Would it be helpful to add a small section with an example of the value for `resources` or a reference link?

Reply:
Ooh, I like it (we should add it to the other config options. How about this:

```suggestion
| `agones.allocator.resources`                        | Allocator pods [resource requests/limit](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)                                                          | `{}`                   |
```

Which also has explanations and examples of resource limits and requests. Sound good?

Reply:
+1 also, I think a simple example would also help with how to set the value. Since it is a compound structure, it is not easy to see how the input should look like.

Reply:
Definitely wouldn't hurt - especially as we have a few config options line this. Maybe add an extra example to:

> Specify each parameter using the --set key=value[,key=value] argument to helm install. For example,

(Or is this something you would have to use a values.yaml for? Trying to find some helm docs we could even point to)

Reply:
As far as I know, resources often uses values.yaml.
I haven't seen many examples of using set to configure it.

What you see most often are commented out settings.

```yaml
resources: {}
  requests:
    # cpu: 1
    # memory: 256Mi
```

Reply:
I've added instructions on how to set the values.